ground/src/main/proto/com/google/ground/job.proto

```diff
@@ -19,11 +19,189 @@ syntax = "proto3";
 option java_package = "com.google.android.ground.proto";
 option java_multiple_files = true;
 
-import "com/google/ground/style.proto";
-
+// A single job, defined by one or more tasks, and zero or more locations of
```

Comment:
> zero or more locations of interest

This is a tad confusing since LOIs are not nested in `Job`, unlike `tasks`. Can you clarify the relationship between LOIs and Jobs?

Reply:
There will be many LOIs in the db, so keeping them in one document wasn't really scalable, whereas there are a small number tasks so nesting under job is ok. Any ideas on how I can represent that here without it seeming too out of context?

Reply:
Maybe something like, "Locations of interest (LOIs) will be associated with the job by the job ID in a separate document" ?

Reply:
Sorry just saw this comment now, I'll address these in the new PR!